storm-core/src/clj/org/apache/storm/ui/core.clj

```diff
@@ -963,11 +963,11 @@
      "uptimeSeconds" uptime
      "host" host
      "port" port
-     "emitted" (Utils/nullToZero (.get_emitted cas))
-     "transferred" (Utils/nullToZero (.get_transferred cas))
+     "emitted" (if-let [em (.get_emitted cas)] em 0)
```

Comment:
But cas is a thrift object and technically emitted is marked as optional, so being defensive I left the nullToZero in place, but I couldn't make nullToZero work with a Double and return a Long decided to just leave the code the same as before, but go to a pure clojure version that will be replaced with core.clj is translated.